### DISCUSSION TITLE
Using Combo Box in VanillaJS

### DISCUSSION THREAD

**Discussion by gemadang** [carbon-design-system/carbon#7576] [Help] [answered]
Hi Can someone post an example of how to use ComboBox in VanillaJS - Can't seem to find any documentation on how to use it anywhere

**Comment by joshblack** [comment, marked as answer]
Hi there, @gemadang! 👋 

Unfortunately, the Vanilla JS version of components have been receiving less attention over the past couple of years. It might be worth checking out the community-driven Web Components project over at: https://github.com/carbon-design-system/carbon-web-components for a similar vanilla-ish feel that is more actively maintained.

Hope this helps!
